support-frontend/assets/pages/subscriptions-redemption/components/redemptionForm.jsx

```diff
@@ -43,27 +41,32 @@ function mapDispatchToProps(dispatch: Dispatch<Action>) {
   };
 }
 
-const InputWithValidated = compose(asControlled, withLabel, withError, withValidation)(Input);
-
 function RedemptionForm(props: PropTypes) {
+  const mainCss = css`
+    padding: ${space[2]}px;
+  `;
   const instructionsDivCss = css`
-    margin-top: -15px;
-    padding: 0 ${space[3]}px;
+    margin-top: -10px;
+    padding: ${space[2]}px;
     ${from.tablet} {
-      min-height: 350px;
+      min-height: 475px;
     }
     hr {
       border: 0;
       border-top: solid 1px ${line.primary};
     }
   `;
-  const paraCss = css`
+  const hrCss = css`
+    margin-bottom: 16px;
+  `;
+  const headingCss = css`
+    ${headline.xsmall({ fontWeight: 'bold' })};
```

Comment:
I am a bit suspicious about how consistently this style of bolding works. Worth checkin on browserstack.